test/integration/azureSpecialProperties.spec.ts

```diff
@@ -275,6 +275,10 @@ describe("AzureSpecialProperties", () => {
           ...clientOptions
         }
       );
+      client.pipeline.removePolicy({
+        name: bearerTokenAuthenticationPolicyName
```

Comment:
These changes are already in main so you should be able to remove these changes in yout PR